ambry-server/src/main/java/com/github/ambry/server/AmbryServerRequests.java

```diff
@@ -95,7 +95,7 @@ public class AmbryServerRequests extends AmbryRequests {
   private final DiskManagerConfig diskManagerConfig;
   private final StatsManager statsManager;
   private final ClusterParticipant clusterParticipant;
-  private final ConcurrentHashMap<RequestOrResponseType, Set<PartitionId>> requestsDisableInfo =
+  private static final ConcurrentHashMap<RequestOrResponseType, Set<PartitionId>> requestsDisableInfo =
```

Comment:
Why do we need to change this to 'static'?

Reply:
Before we use the single AmbryServerRequests object, so whenever there is a adminCommand to change the requestsDisableInfo, it applies to this single object. And all the handlers are aware of that.
After we change to use one object for each handler, all the handler should be aware of it as well. If we don't use static, only one thread is aware of the requestsDisableInfo.